requirements/static/ci/py3.5/darwin.txt

```diff
@@ -220,6 +220,7 @@ timelib==0.2.5
 toml==0.10.2
 transitions==0.8.1        # via junos-eznc
 urllib3==1.24.3
+validators==0.18.2
```

Comment:
I prefer we don't add a dependency for validating URLs.

Reply:
urlparse doesn't actually really validate URLs, otherwise I'd have used it.

```
>>> from urllib.parse import urlparse
>>> url = "http://netbox.example.com;"
>>> result = urlparse(url)
>>> print(result)
ParseResult(scheme='http', netloc='netbox.example.com;', path='', params='', query='', fragment='')
>>> url = "http:///netbox.example.com"
>>> result = urlparse(url)
>>> print(result)
ParseResult(scheme='http', netloc='', path='/netbox.example.com', params='', query='', fragment='')
```

I can live with dropping the URL validation altogether, as I knew adding a dependency for it might be a tough sell. I was just trying to improve the user experience (especially in light of #59398), as I know when I started out with Salt, having a bunch of cryptic python exceptions spat out wasn't very helpful.

Reply:
I've pushed commit 91ed2518f122089c0942bd6c3f13756c88fcd08f and 
commit 7b3bab92c93324a5ac65741ce34b9bef92459e82  to address.

Reply:
@dwoz does this adequately address your concerns?